tldw_Server_API/app/api/v1/endpoints/media/process_emails.py

```
@@ -336,6 +359,77 @@ async def _email_batch_processor(
+                        template=chunk_options_dict.get("hierarchical_template")
+                        if isinstance(
+                            chunk_options_dict.get("hierarchical_template"), dict
+                        )
+                        else None,
+                    )
+                else:
+                    chunks = _improved_chunking_process(text, chunk_options_dict)
+
+                res["chunks"] = chunks
+    except Exception:
+        pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)